package.json

```diff
@@ -40,7 +40,7 @@
     "url": "git+https://github.com/ioBroker/ioBroker.zwave.git"
   },
   "dependencies": {
-    "openzwave-shared": "^1.5.5",
+    "openzwave-shared": "^1.5.6",
```

Comment:
This is unnecessary, since `^1.5.5` also includes version 1.5.6. But it doesn't hurt either.